v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/endtoend/EndToEndTestingITBase.java

```diff
@@ -362,4 +598,122 @@ protected JDBCSource createMySqlDatabase(
             Map.of(cloudSqlResourceManager.getDatabaseName(), tables.keySet().stream().toList()))
         .build();
   }
+
+  protected String generateSessionFile(
```

Comment:
Can you rename this to `generateSessionFileUsingSMT` ? And the generateSessionFile method at Line 579 should be renabled to `uploadSessionFile` .